src/libs/DomUtils/index.ts

```diff
@@ -0,0 +1,16 @@
+import {BlurActiveElement, GetActiveElement} from './types';
+
+const blurActiveElement: BlurActiveElement = () => {
+    const activeElement = document.activeElement as HTMLElement;
+
+    if (!activeElement?.blur) return;
```

Comment:
This should be like so, right?
```suggestion
    if (!activeElement?.blur) {
        return;
    }
```

Reply:
Should be ok now @cead22